packages/apps/transfer/.eslintrc.js

```diff
@@ -1,8 +1,32 @@
+const fs = require('fs');
+
+const prettierOptions = JSON.parse(fs.readFileSync('./.prettierrc', 'utf8'));
 require('@rushstack/eslint-config/patch/modern-module-resolution');
 
 module.exports = {
-  plugins: ['jest'],
+  plugins: ['prettier', 'jest'],
   extends: ['next/core-web-vitals', '@kadena-dev/eslint-config/profile/react'],
   parserOptions: { tsconfigRootDir: __dirname },
   ignorePatterns: ['**/*.spec.ts', '**/*.spec.tsx', '**/generated/**'],
+  rules: {
+    'prettier/prettier': ['error', prettierOptions],
+    '@typescript-eslint/strict-boolean-expressions': 0,
+    'arrow-body-style': [2, 'as-needed'],
```

Comment:
simplify arrow function and use parentheses instead of return keyword when function only returns 
e.g.
```
// turns this
const foo = () => {
    return "bar";
}
// to this
const foo = () => "bar"
```

Reply:
hm would prefer to be free in your choice with this